src/Test/CountryInfo.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../Redux/configStore';
+
+import CountryInfo from '../Components/Countries/CountryInfo';
+
```

Comment:
- [ ] It would be better if we write more tests to test the functionality of the app, remember to add negative test cases and also write test for our redux pure functions